sphinx_performance/analysis.py

```diff
@@ -251,16 +253,29 @@ def cli_analysis(
             show_all = False
             processor_options["filter_threshold"] = tree_filter
 
-        html_data = HTMLRenderer(
+        json_str = JSONRenderer(
             show_all=show_all,
             processor_options=processor_options,
         ).render(
             all_profile,
         )
-        with Path.open("pyinstrument_profile.html", "w") as profile_html:
-            profile_html.write(html_data)
+        html_data = HTMLRendererFromJson(
+            show_all=show_all,
+            processor_options=processor_options,
+        ).render(
+            json_str,
+        )
+        with Path.open(Path("pyinstrument_profile.html"), "w") as events_json_file:
```

Comment:
```suggestion
        with Path("pyinstrument_profile.html").open("w") as events_json_file:
```

Reply:
Also this path should probably be absolute, otherwise you are relying on the current context?

Reply:
we have 2 options
- integration to /tmp dir where the build happens
- store the CWD
Of those 2 I prefer CWD as /tmp has to be located first and it changes all the tiem.